pipeline/tasks/task-scripts/prepare-protected-site.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+appFile=${1:-"ci-app.yaml"}
+spProtocol=${SP_PROTOCOL:-"https"}
+
+# SSO Protection
+if [[ "${PROTECTED}" == "true" ]] ; then
+  if [[ ! -f ${appFile} ]] ; then
+    cp app.yaml ${appFile}
+  fi
+  echo "Enhancing ${appFile} with env_variables"
+  mkdir -p .deploy/idp
+  mkdir -p .deploy/sp
+
+  set +x
+  mkdir -p .build
+  echo ${IDP_CERTIFICATE} > .build/encoded-idp-certificate.txt
+  echo ${SP_CERTIFICATE} > .build/encoded-sp-certificate.txt
+  echo ${SP_KEY} > .build/encoded-sp-key.txt
+  set -x
+
+  if [[ `base64 --help | grep "hvD" | wc -l` -gt 0 ]] ; then
```

Comment:
use `uname` instead?
https://stackoverflow.com/a/3466183

Reply:
ok - that's better - done